main.py

```diff
@@ -1,7 +0,0 @@
-# Import function here and call it below, e.g.
-# from aws.database ... import my_script
-
-if __name__ == "__main__":
-    # 1) Edit and configure scripts (including stage) in source files
-    # 2) Call functions here, e.g my_script()
-    pass
```

Comment:
So I assume, from now on I just navigate to specific script and click the VS Code play button on that rather than doing this main script thing?